### PR TITLE
Buf.9 - remove commented out .Xr reference to <fillmein>(9)

### PR DESCRIPTION
**Pull request by chrisdavidson**
buf.9 - Update to manual page to remove commented out .Xr to <filmmein>(9).

### CODE REVIEW

File: share/man/man9/buf.9 [hunk 1]
```diff
@@ -132,8 +132,7 @@ instantiated VM Buffers (struct buf's) prevent their underlying pages in the
 buffer cache from being freed.
 This can complicate the life of the paging
 system.
-.\" .Sh SEE ALSO
-.\" .Xr <fillmein> 9
+
```
Comment (bsdimp) [inline share/man/man9/buf.9:135]: Likely don't want a blank line here

File: sys/contrib/openzfs/man/man8/zpool-iostat.8 [hunk 2]
```diff
@@ -146,7 +146,11 @@ Specify
 .Sy u
 for a printed representation of the internal representation of time.
 See
+<<<<<<< HEAD
 .Xr time 1 .
+=======
+.Xr time 2 .
+>>>>>>> 99007677d048 (Revert "elftc_set_timestamps.3 modification to update from utime(2) -> utime(3)")
```
Comment (bsdimp) [inline sys/contrib/openzfs/man/man8/zpool-iostat.8:153]: Merge conflict

File: sys/contrib/openzfs/man/man8/zpool-list.8 [hunk 3]
```diff
@@ -95,7 +95,11 @@ Specify
 .Sy u
 for a printed representation of the internal representation of time.
 See
+<<<<<<< HEAD
 .Xr time 1 .
+=======
+.Xr time 2 .
+>>>>>>> 99007677d048 (Revert "elftc_set_timestamps.3 modification to update from utime(2) -> utime(3)")
```
Comment (bsdimp) [inline sys/contrib/openzfs/man/man8/zpool-list.8:102]: Merge conflicts

File: sys/contrib/openzfs/man/man8/zpool-status.8 [hunk 4]
```diff
@@ -112,7 +112,11 @@ Specify
 .Sy u
 for a printed representation of the internal representation of time.
 See
+<<<<<<< HEAD
 .Xr time 1 .
+=======
+.Xr time 2 .
+>>>>>>> 99007677d048 (Revert "elftc_set_timestamps.3 modification to update from utime(2) -> utime(3)")
```
Comment (bsdimp) [inline sys/contrib/openzfs/man/man8/zpool-status.8:119]: Merge conflict

File: sys/contrib/openzfs/man/man8/zpool-wait.8 [hunk 5]
```diff
@@ -99,7 +99,11 @@ Specify
 .Sy u
 for a printed representation of the internal representation of time.
 See
+<<<<<<< HEAD
 .Xr time 1 .
+=======
+.Xr time 2 .
+>>>>>>> 99007677d048 (Revert "elftc_set_timestamps.3 modification to update from utime(2) -> utime(3)")
```
Comment (bsdimp) [inline sys/contrib/openzfs/man/man8/zpool-wait.8:106]: Merge conflict

File: share/man/man4/lagg.4 [hunk 6]
```diff
@@ -214,7 +214,7 @@ The following example shows how to create an infiniband failover interface.
 .Ed
 .Pp
 Configure two ethernets for failover with static IP in
-.Xr /etc/rc.conf 5 :
+/etc/rc.conf:
```
Comment (bsdimp) [inline share/man/man4/lagg.4:217]: At least do .Pa here.
But i think there is rc.conf man page....

Comment (chrisdavidson) [reply share/man/man4/lagg.4:217]: okay, there is a reference to rc.conf(5) in the SEE ALSO section, as it was currently written.. it was incorrectly referencing a /etc/rc.conf 5 that did NOT exist.

File: share/man/man9/buf.9 (hunk 1 above)
Comment (chrisdavidson) [reply share/man/man9/buf.9]: blank line address in subsequent commit.

File: share/man/man4/lagg.4 (hunk 6 above)
Comment (chrisdavidson) [reply share/man/man4/lagg.4:217]: addressed this and added the .Pa macro to the line, confirmed with mantra utility to be operational.

**Comment by chrisdavidson** [comment]
i need to re-evaluate my git workflow process and as such i am going to close out these pull request, till i resolve those issues.

**Pull request closed without merging**